hardhat/test/player.js

```diff
@@ -0,0 +1,76 @@
+const { ethers } = require('hardhat');
+const { use, expect } = require('chai');
+const { solidity } = require('ethereum-waffle');
+
+use(solidity);
+
+describe('Player', function () {
+  let sharShip;
+  let player;
+  let world;
+  const worldId = 1;
+
+  it('should deploy sharShip', async function () {
+    const YourContract = await ethers.getContractFactory('Starship');
+
+    sharShip = await YourContract.deploy();
+  });
+  it('should deploy Players', async function () {
+    const YourContract = await ethers.getContractFactory('Players');
+
+    player = await YourContract.deploy();
+    await player.setNftAddress(sharShip.address);
+  });
```

Comment:
is there a reason variable is sharShip as opposed to starship? (here and elsewhere)